doc/rados/operations/control.rst

```diff
@@ -68,7 +68,7 @@ execute the following::
 	ceph pg dump_stuck inactive|unclean|stale|undersized|degraded [--format {format}] [-t|--threshold {seconds}]
 
 
-``--format`` may be ``plain`` (default) or ``json``
+``--format`` may be ``plain`` (default), ``json``, ``json-pretty``, ``xml``, or ``xml-pretty``.
```

Comment:
There are other places in this doc where we aren't listing all the options. Might be useful to add " --format {json,json-pretty,xml,xml-pretty,plain}" in all those places or in one common place.

Reply:
I've taken this on board, and added the full list where it appears in the doc. "pg dump" also appears in rados/operations/monitoring-osd-pg.rst, but the context doesn't cry out for the exhaustive list of formats.